### PR TITLE
Support

### PR DESCRIPTION
**Pull request by crawfordsm**
This adds a section on financial support.   Please feel free to correct anything I got wrong in here.  

Pushing this here for discussion.   Should only be merged with approval from the coordination committee ( @eteq @astrofrog @kelle @taldcroft ) but open for everyone to discuss.

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -369,7 +369,26 @@ \subsection{Distribution of packages}
 %
 % ^.
 
-%\subsection{Support of Astropy}
+\subsection{Support of Astropy}
+
+The \astropy project, as of the v2.0 release, does not receive any direct financial support for
+the development of the \astropypkg.  Development of the \astropypkg is supported
+through contributions by individuals to work on astropy in their own personal
```
Comment (astrofrog) [inline main.tex]: by individuals *who* work on astropy? (same in following clause)

File: main.tex [hunk 2]
```diff
@@ -369,7 +369,26 @@ \subsection{Distribution of packages}
...
+time, contributions from individuals or groups to work on \astropy as 
+part of a research project, or contributions from institutes that allocate 
+people to work on \astropy. 
+A list of organizations that have contributed to \astropy
+are listed in the Acknowledgements.  
+Different models have been proposed for the support of astropy
```
Comment (astrofrog) [inline main.tex]: support of astropy *in future*?

File: main.tex [hunk 3]
```diff
@@ -369,7 +369,26 @@ \subsection{Distribution of packages}
...
+the development of the \astropypkg.  Development of the \astropypkg is supported
+through contributions by individuals to work on astropy in their own personal
+time, contributions from individuals or groups to work on \astropy as 
+part of a research project, or contributions from institutes that allocate 
+people to work on \astropy. 
+A list of organizations that have contributed to \astropy
+are listed in the Acknowledgements.  
+Different models have been proposed for the support of astropy 
+(ie. \cite{2016arXiv161003159M}), 
+but a sustainable, community-based model has not yet been established.
```
Comment (astrofrog) [inline main.tex]: It's not clear to me what *community-based model* means here - what about instead saying 'but a long-term plan for sustainability has not yet been established' or something like this? I may be misunderstanding this though.

File: main.tex [hunk 4]
```diff
@@ -369,7 +369,26 @@ \subsection{Distribution of packages}
...
+The astropy project has the ability to accept financial contributions
```
Comment (astrofrog) [inline main.tex]: astropy -> Astropy

File: main.tex [hunk 5]
```diff
@@ -369,7 +369,26 @@ \subsection{Distribution of packages}
...
+are listed in the Acknowledgements.  
+Different models have been proposed for the support of astropy 
+(ie. \cite{2016arXiv161003159M}), 
+but a sustainable, community-based model has not yet been established.
+The astropy project has the ability to accept financial contributions 
+(individual or institutes) through the NumFOCUS organization\footnote{NumFOCUS 
+is a 501(c)(3) nonprofit that supports and promotes world-class, innovative, 
+open source scientific computing.},  
+which generally covers the direct costs incurred by the \astropy project
+along with supporting
```
Comment (astrofrog) [inline main.tex]: missing text at the end?

**Review by astrofrog — COMMENTED** [review]
Looks good! Just a few comments :)

File: main.tex [hunk 6]
```diff
@@ -369,7 +369,25 @@ \subsection{Distribution of packages}
...
+The \astropy project, as of the v2.0 release, does not receive any direct financial support for
+the development of the \astropypkg.  Development of the \astropypkg is supported
+by individuals who work on \astropy project in their own personal
+time, by individuals or groups to work on \astropy as 
+part of a research project, or contributions from institutes that allocate 
+people to work on \astropy. 
+A list of organizations that have contributed to \astropy
+are listed in the Acknowledgements.  
+Different models have been proposed for support of \astropy in the future 
+(ie. \cite{2016arXiv161003159M}), but a long-term plan
```
Comment (astrofrog) [inline main.tex]: ie. -> e.g.?

**Review by astrofrog — APPROVED** [review]
Looks good to me but I think @kelle, @eteq and @taldcroft should probably review this (at least one of them)

File: main.tex [hunk 7]
```diff
@@ -369,7 +369,25 @@ \subsection{Distribution of packages}
...
 % ^.
 
-%\subsection{Support of Astropy}
+\subsection{Support of Astropy}
+
+The \astropy project, as of the v2.0 release, does not receive any direct financial support for
+the development of the \astropypkg.  Development of the \astropypkg is supported
+by individuals who work on \astropy project in their own personal
+time, by individuals or groups to work on \astropy as 
+part of a research project, or contributions from institutes that allocate
```
Comment (eteq) [inline main.tex]: "institutes" to "institutions" (the latter seems slightly more general to me)

File: main.tex [hunk 8]
```diff
@@ -369,7 +369,25 @@ \subsection{Distribution of packages}
...
-%\subsection{Support of Astropy}
+\subsection{Support of Astropy}
+
+The \astropy project, as of the v2.0 release, does not receive any direct financial support for
+the development of the \astropypkg.  Development of the \astropypkg is supported
+by individuals who work on \astropy project in their own personal
+time, by individuals or groups to work on \astropy as 
+part of a research project, or contributions from institutes that allocate 
+people to work on \astropy. 
+A list of organizations that have contributed to \astropy
```
Comment (eteq) [inline main.tex]: maybe "contributed to astropy in this manner"?  (i.e. right now it reads like "no one has contributed to astropy.  organizations that contribute to astropy are listed in the acknowledgements.

File: main.tex [hunk 9]
```diff
@@ -369,7 +369,25 @@ \subsection{Distribution of packages}
...
+are listed in the Acknowledgements.  
+Different models have been proposed for support of \astropy in the future
```
Comment (eteq) [inline main.tex]: models -> "funding models"?

**Review by eteq — COMMENTED** [review]
Left a few wordsmithing comments, but overall I'm happy with this as a way to dance around some of the tricky political phrasing questions.

**Comment by adrn** [comment]
I made a few more nitpicky edits, but LGTM otherwise. Merge when tests pass.

**Pull request merged**